### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jennamk14**
added link to wiki for issue #61 
Added page to wiki on database https://github.com/Imageomics/kabr-tools/wiki/Data-Management-Tips

### CODE REVIEW

**Comment by jennamk14** [comment]
@egrace479 Added these revisions based on feedback from code review 
@zhong-al fyi

**Comment by zhong-al** [comment]
@jennamk14

The links for rename.sh & downgrade.sh link to creating a new wiki page. There's a similar problem on the CVAT User Guide page - the links to tracks_extractor.py and time_budget.ipynb don't work.

**Comment by egrace479** [comment]
> @jennamk14
> 
> The links for rename.sh & downgrade.sh link to creating a new wiki page. There's a similar problem on the CVAT User Guide page - the links to tracks_extractor.py and time_budget.ipynb don't work.

I updated all 4 the links in the wiki. I adjusted the tracks extractor one to point to the method instructions instead of the file. @jennamk14, relative links don't work to the repo from the wiki--it's managed separately. In fact, you can actually clone a wiki which creates `<repo-name>.wiki` and has full version control tracking through the command line as well.

**Comment by jennamk14** [comment]
> > @jennamk14
> > The links for rename.sh & downgrade.sh link to creating a new wiki page. There's a similar problem on the CVAT User Guide page - the links to tracks_extractor.py and time_budget.ipynb don't work.
> 
> I updated all 4 the links in the wiki. I adjusted the tracks extractor one to point to the method instructions instead of the file. @jennamk14, relative links don't work to the repo from the wiki--it's managed separately. In fact, you can actually clone a wiki which creates `<repo-name>.wiki` and has full version control tracking through the command line as well.

ok thanks for the info! how should be point users from the readme to the wiki?

**Comment by egrace479** [comment]
> > > @jennamk14
> > > The links for rename.sh & downgrade.sh link to creating a new wiki page. There's a similar problem on the CVAT User Guide page - the links to tracks_extractor.py and time_budget.ipynb don't work.
> > 
> > 
> > I updated all 4 the links in the wiki. I adjusted the tracks extractor one to point to the method instructions instead of the file. @jennamk14, relative links don't work to the repo from the wiki--it's managed separately. In fact, you can actually clone a wiki which creates `<repo-name>.wiki` and has full version control tracking through the command line as well.
> 
> ok thanks for the info! how should be point users from the readme to the wiki?

The current pointers are good; using the full URL works.

File: README.md [hunk 1]
```diff
@@ -48,9 +48,11 @@ Recognition from Drone Videos](https://openaccess.thecvf.com/content/WACV2024W/C
...
 In order to automatically label the animal videos with behavior, we must first create *mini-scenes* of each individual animal captured in the frame, illustrated below.
 
+See the Wiki [CVAT User Guide](https://github.com/Imageomics/kabr-tools/wiki/CVAT-User-Guide) and [Data Management Tips](https://github.com/Imageomics/kabr-tools/wiki/Data-Management-Tips) for detailed instructions and recommendations.
+
 ![](images/im_mini-scenes.jpg)
-**Figure 3:** A mini-scene is a sub-image cropped from the drone video footage centered on and surrounding a single animal. Mini-scenes simulate the camera as well-aligned with each individual animal in
-the frame, compensating for the movement of the drone and ignoring everything in the large field of
+**Figure 3:** A mini-scene is a sub-image cropped from the drone video footage centered on and surrounding a single animal. Mini-scenes simulate the camera as well-aligned with each animal in
+the frame, compensating for the drone's movement and ignoring everything in the large field of
 view but the animal’s immediate surroundings. The KABR dataset consists of mini-scenes and their
```
Comment (egrace479) [inline README.md:56]:
```suggestion
the frame, compensating for the drone's movement by focusing on just the animal and its immediate surroundings. The KABR dataset consists of mini-scenes and their
```
Would it be more appropriate to describe it as "and focusing..." or "by focusing..." I think that focus is at least part of how it compensates for the drone movement?

Comment (jennamk14) [reply README.md:56]: yes, i like 'focusing' a lot better here

**Review by egrace479 — APPROVED** [review]
I just made the text change. We should also make sure that the sample images have alt-text (it goes between the brackets "[]"), that can be done with a separate issue.

**Pull request merged**